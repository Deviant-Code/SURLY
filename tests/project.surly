PRINT Student;

P1 = PROJECT sid, name, FROM Student;
PRINT P1;

P2 = PROJECT COUNT(gpa), AVG(gpa) FROM Student;
PRINT P2;

