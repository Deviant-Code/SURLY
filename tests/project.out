
Printing 1 relation(s):
Student:
sid  name              gpa   
  0  Malcolm Reynolds   2.4  
  1  Zoe Washburne     3.35  
  2  Wash Washburne    2.85  
  3  Inara Serra       3.65  
  4  Jayne Cobb        0.35  
  5  Kaylee Frye        3.3  
  6  Simon Tam          3.9  
  7  River Tam            4  
  8  Derrial Book      3.75  

Assigning variable 'P1'
Printing 1 relation(s):
P1:
sid  name              
  0  Malcolm Reynolds  
  1  Zoe Washburne     
  2  Wash Washburne    
  3  Inara Serra       
  4  Jayne Cobb        
  5  Kaylee Frye       
  6  Simon Tam         
  7  River Tam         
  8  Derrial Book      

Assigning variable 'P2'
Printing 1 relation(s):
P2:
gpa  gpa     
  9  3.0611  
